kernel/configs/fragments/4.19.x/common/virtio.conf

```diff
@@ -0,0 +1,23 @@
+# We need virtio for 9p and serial and vsock at least
+
+# To get VIRTIO, we need a bus - ours of choice is PCI. We need to enable
+# PCI support to get VIRTIO_PCI support
+CONFIG_PCI=y
+CONFIG_PCI_MSI=y
+CONFIG_PCI_MSI_IRQ_DOMAIN=y
+# To get to the VIRTIO_PCI, we need the VIRTIO_MENU enabled
+CONFIG_VIRTIO_MENU=y
+CONFIG_VIRTIO_PCI=y
```

Comment:
Have we thought/talked about a separate config for firecracker. I would think we can trim down this list even further,

Reply:
We could - it would be easy to try and make the tree take into account different VMs as well - I think the existing code (for full config files) already had some notion of that, but:
- will we be having our own FC kernel, or use the one that comes with FC?
- at the moment we only ship one kernel I think, so the present setup is set to build one default kernel per architecture. I have thought we may add some extra fragment dirs/hierarchy so we can have a set of extra config options like minimal/maximal/debug etc.

So, I think it will need a little more discussion on how many different kernel configs we want to support and ship.

Reply:
oh, and for ref, afaik the FC kernel config is [here](https://github.com/firecracker-microvm/firecracker/blob/7f29bca9ca197283275eab62fddc1c10ab580794/resources/microvm-kernel-config), and I think when I did a count of the `=y` settings in there, they had nearly as many as our current config (~1000), which is more than this current frag config.